build-scripts/bundle.js

```diff
@@ -53,14 +53,16 @@ module.exports.definedVars = ({ isProdBuild, latestBuild, defineOverlay }) => ({
   ...defineOverlay,
 });
 
-module.exports.htmlMinifierOptions = {
+const htmlMinifierOptions = {
```

Comment:
I purposely exported it because I planned to migrate to use it for the HTML templates too 😉

Reply:
haha you can export it again when you need it 😉